tests/Backend/Snowflake/TableDefinitionOperationsTest.php

```diff
@@ -77,6 +79,712 @@ public function testResponseDefinition(): void
+                    ],
+                ],
+            ],
+            'DOUBLE' => [
+                'working' => [
+                    'value' => '1',
+                    'expectFail' => [],
+                ],
+                'working_num' => [
+                    'value' => 1,
+                    'expectFail' => [],
+                ],
+                'empty' => [
+                    'value' => '',
+                    'expectFail' => [],
+                ],
+                'fail type' => [
+                    'value' => 'string',
+                    'expectFail' => [
+                        'message' => 'Table creation ended with a syntax exception, probably due to an invalid "default" column specification. Original exception is:',
+                    ],
+                ],
+            ],
+            'DOUBLE PRECISION' => [
+                'working' => [
+                    'value' => '1',
+                    'expectFail' => [],
+                ],
+                'working_num' => [
+                    'value' => 1,
+                    'expectFail' => [],
+                ],
+                'empty' => [
+                    'value' => '',
+                    'expectFail' => [],
+                ],
+                'fail type' => [
+                    'value' => 'string',
+                    'expectFail' => [
+                        'message' => 'Table creation ended with a syntax exception, probably due to an invalid "default" column specification. Original exception is:',
+                    ],
+                ],
+            ],
+            'REAL' => [
+                'working' => [
+                    'value' => '1',
+                    'expectFail' => [],
+                ],
+                'working_num' => [
+                    'value' => 1,
+                    'expectFail' => [],
+                ],
+                'empty' => [
+                    'value' => '',
+                    'expectFail' => [],
+                ],
+                'fail type' => [
+                    'value' => 'string',
+                    'expectFail' => [
+                        'message' => 'Table creation ended with a syntax exception, probably due to an invalid "default" column specification. Original exception is:',
+                    ],
+                ],
+            ],
+            'VARCHAR' => [
+                'working' => [
+                    'value' => '\'test\'',
+                    'expectFail' => [],
+                ],
+                'empty' => [
+                    'value' => '',
+                    'expectFail' => [],
+                ],
+                'not quoted' => [
+                    'value' => 'test',
+                    'expectFail' => [],
+                ],
+                'wrong quoted' => [
+                    'value' => '"test"',
+                    'expectFail' => [],
```

Comment:
Tohle by mělo spadnout, ne? To prostě není správný string - je to jako kdyby uživatel chtěl zadat nějaký jiný sloupec jako default

Reply:
```
CREATE OR REPLACE TABLE mytable (id boolean DEFAULT "test", cas TIME DEFAULT '00:00:00'::time, num number);
[42000][904] SQL compilation error: error line 1 at position 52 invalid identifier '"test"'
```

Reply:
s varchar column to dělá stejně

Reply:
CREATE OR REPLACE TABLE mytable (id boolean DEFAULT '"test"', cas TIME DEFAULT '00:00:00'::time, num number);